go/client/cmd_ctl.go

```diff
@@ -36,15 +34,15 @@ func NewCmdCtl(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
 }
 
 // availableComponents specify which components can be included or excluded
-var availableCtlComponents = []string{
+var availableCtlComponents = []string{ //nolint
```

Comment:
just curious: why the `nolint` for these?

Reply:
They are only used when the build tag is `darwin`, and so seem unused when linting on Linux.  I guess we could move them into a darwin-only file but this was faster to do at the time.